client/src/docs-ui/footer/footer.tsx

```diff
@@ -8,6 +8,7 @@ import {
   socialLinkContainerStyle,
 } from "./footer.style";
 import * as links from "../../constants/links";
+import * as img from "../../constants/img";
```

Comment:
Looks like we do individual imports in most places. Let's keep it consistent and do it here as well